### ISSUE TITLE
File encoding

### ISSUE THREAD

**Issue by diegohavenstein** [tommybennett/algorithm-mnemonics#4]
Hi Tommy!

(Great talk at CppCon!)

Wanted to try it out on my linux machine today. Do you think it would be possible to define the leader key centrally for those nasty escape sequences? (not sure how the file looks on DOS) Or choosing a different encoding?

cpp_abbr.vim:       C source, ASCII text
cpp_algorithms.vim: ASCII text, with CRLF, CR, LF line terminators, with escape sequences
cpp_funcs.vim:      ASCII text, with CRLF, CR, LF line terminators, with escape sequences
cpp_map.vim:        ASCII text, with escape sequences
cpp_util.vim:       ASCII text
README:             ASCII text

What do you think?

Thanks for the great idea!

Regards
Diego

**Comment by tommybennett** [comment]
Hello Diego,

I'm glad you liked the talk.  Yes, the vim implementation was hard to read with the way the escape sequences were encoded.  I changed it to use _execute normal!_ which recognizes the characters _<esc>_ for escape and _<cr>_ for carriage return.

I have committed a new version that removes the escape sequences.  Try it out and let me know if you have any other issues.

Thank you for your suggestion.

Regards,
Tommy
